src/todos.js

```diff
@@ -0,0 +1,194 @@
+    if (tasks) {
+      tasks.sort((a, b) => a.index - b.index); // Sort Tasks based on their Index Value
+      document.getElementById('tasks').innerHTML = '';
+      tasks.forEach((task) => {
+        this.add(task);
+      });
+    } else {
+      const taskList = document.getElementById('tasks');
+      taskList.style.display = 'none'; // Remove Shadow if there are no tasks
+    }
+    // Give Ability To Edit Recently added Todos
+    this.editTodos();
+    this.checkTodo();
+    this.editMode();
+  }
+
+  // Give a dashed style To todos when checked
+  static checkTodo() {
+    const todos = document.querySelectorAll('.task');
+    todos.forEach((todo) => todo.addEventListener('click', (e) => {
+      if (e.target.classList.contains('check')) {
+        const des = e.target.nextSibling;
+        let todos = Store.getTodos();
+        // When Todo is Checked
+        if (e.target.checked) {
+          des.style.textDecoration = 'line-through';
+          des.style.color = 'grey';
+          const checked = todos.filter((todo) => todo.description === des.innerText);
+          checked[0].completed = true;
+          const rest = todos.filter((todo) => todo.description !== des.innerText);
+          todos = [...checked, ...rest];
+          todos.sort((a, b) => a.index - b.index);
+        } else { // When Todo is Uncheked
+          des.style.textDecoration = 'none';
+          des.style.color = '#003b46';
+          const checked = todos.filter((todo) => todo.description === des.innerText);
+          checked[0].completed = false;
+          const rest = todos.filter((todo) => todo.description !== des.innerText);
+          todos = [...checked, ...rest];
+          todos.sort((a, b) => a.index - b.index);
+        }
+        // Update Store to Checked or Not Checked
+        Store.setTodos(todos);
+      }
+    }));
+  }
+
+  // Edit Todos When Clicked
+  static editTodos() {
+    const todos = document.querySelectorAll('.task');
+    todos.forEach((todo) => {
+      // Listen For Clicks On Todos
+      todo.addEventListener('click', (e) => {
+        if (!e.target.classList.contains('check')) {
+          const target = e.target.closest('.task');
+
+          // Add Edit Classes On Clicked Todos
+          if (!target.classList.contains('edit')) {
+            // Reset all Todo Styles
+            todos.forEach((todo) => {
+              todo.classList.remove('edit');
+              todo.children[2].remove();
+              const dots = document.createElement('i');
+              dots.className = 'fa-solid fa-ellipsis-vertical dots';
+              todo.append(dots);
+            });
+
+            // Apply Todo Styles
+            target.classList.toggle('edit');
+            target.children[2].remove();
+            const bin = document.createElement('i');
+            bin.className = 'fa-solid fa-trash-can';
+            target.append(bin);
+            bin.addEventListener('click', Store.deleteTodo);
```

Comment:
- [ ] [Optional] Please fix editing todo task functionality because It's not running properly.
### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️